app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split(" ")
+    file1 = command_list[1]
+    file2_path = command_list[-1]
+    if "/" in command:
+        directories = command_list[-1].split("/")[:-1]
+        file_path = ""
+        for directory in directories:
+            file_path += f"{directory}/"
+            if not os.path.isdir(file_path):
+                os.mkdir(file_path)
```

Comment:
Code Efficiency: Notice that if you try to create a directory that already exists, it will lead to an error. Consider handling this case to avoid potential issues.